docs/source/resources/define-metadata.rst

```diff
@@ -1000,6 +1000,23 @@ Package names should follow the `package match specifications <https://conda.io/
        - optional-subpackage =={{ version }}
 
 
+For example, let's say we have an environment that has package "a" installed at
+version 1.0. If we install package "b" that has a run_constrained entry of
+"a>1.0", then conda would need to upgrade "a" in the environment in order to
+install "b".
+
+This is especially useful in the context of virtual packages, where the
+`run_constrained` dependency is not a package that conda manages, but rather a
```

Comment:
Is it wrong for a virtual package to be a `run` dependency?

Reply:
It is not wrong to have a virtual package as a `run` dependency but it does limit what versions of conda can be used to install the package.  Versions of conda that pre-date the addition of the virtual package will not able to install a package with a `run` dependency but will be able to install a package with a `run_constrained` dependency.